curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c9c0d0857f0a10a57af936.md

```diff
@@ -38,13 +38,13 @@ assert.match(calculateCalories.toString(), /snacksNumberInputs\s*=/);
 You should call `document.querySelectorAll()` with `#snacks input[type=number]`.
 
 ```js
-assert.match(calculateCalories.toString(), /document\.querySelectorAll\(\s*('|")#snacks input\[\s*type\s*=\s*number\s*\]\s*\1\s*\)/);
+assert.match(calculateCalories.toString(), /document\.querySelectorAll\(\s*('|"|`)#snacks input\[\s*type\s*=\s*('|")?number('|")?\s*\]\s*\1\s*\)/);
 ```
 
 You should assign the result of your `document.querySelectorAll()` call to `snacksNumberInputs`.
 
 ```js
-assert.match(calculateCalories.toString(), /snacksNumberInputs\s*=\s*document\.querySelectorAll\(\s*('|")#snacks input\[\s*type\s*=\s*number\s*\]\s*\1\s*\)/);
+assert.match(calculateCalories.toString(), /snacksNumberInputs\s*=\s*document\.querySelectorAll\(\s*('|"|`)#snacks input\[\s*type\s*=\s*('|")?number('|")?\s*\]\s*\1\s*\)/);
```

Comment:
```suggestion
assert.match(calculateCalories.toString(), /snacksNumberInputs\s*=\s*document\.querySelectorAll\(\s*('|"|`)#snacks input\[\s*type\s*=\s*('|")?number\2?\s*\]\s*\1\s*\)/);
```
use a capture group to match the same type of quote, this needs to be changed in all the tests